src/_layouts/default.html

```diff
@@ -7,4 +7,6 @@
   <body>
     {% include header.html %} {{content}}
   </body>
+
+  {% include footer.html %}

```

Comment:
Nice idea to make this an include! I think it should be inside the `<body>` tag though, right?

Reply:
Here's a good explanation of why: https://stackoverflow.com/a/12107279/453168